doc/ref_cert/RC2/chapters/chapter02.rst

```diff
@@ -0,0 +1,611 @@
+must run in order to test compliance with the requirements. This will
+enable traceability between the test cases and requirements. They should
+be able to clearly see which requirements are covered by which tests and
+the mapping from a specific test result (pass or fail) to a requirement.
+Each requirement may have one or more test case associated with it.
+
+Goals
+~~~~~
+
+-  Clear mapping between requirements and test cases
+-  Provide a stable set of point-in-time requirements and tests to
+   achieve conformance
+-  Enable clear traceability of the coverage of requirements across
+   consecutive releases of this document
+-  Clickable links from test cases to requirements
+-  One or more tests for every MUST requirement
+-  A set of test cases to serve as a template for Anuket Assured
+
+Non-Goals
+~~~~~~~~~
+
+-  Defining any requirements
+-  Providing coverage for non-testable requirements
+
+Definitions
+~~~~~~~~~~~
+
+*must*: Test Cases that are marked as must are considered mandatory and
+must pass successfully
+
+*should*: Test Cases that are marked as should are expected to be
+fulfilled by the cloud infrastructure but it is up to each service
+provider whether to accept a cloud infrastructure that is not fulfilling
+any of these requirements. The same applies to should not.
+
+*may*: Test cases that are marked as may are considered optional. The
+same applies to may not.
+
+Traceability Matrix
+-------------------
+
+Kubernetes API testing
+~~~~~~~~~~~~~~~~~~~~~~
+
+The primary objectives of the `e2e
+tests <https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md>`__
+are to ensure a consistent and reliable behavior of the Kubernetes code
+base, and to catch hard-to-test bugs before users do, when unit and
+integration tests are insufficient. They are partially selected for the
+`Software Conformance Certification
+program <https://github.com/cncf/k8s-conformance>`__ run by the
+Kubernetes community (under the aegis of the CNCF).
+
+Anuket shares the same goal to give end users the confidence that when
+they use a certified product they can rely on a high level of common
+functionality. Then Anuket RC2 starts with the test list defined by `K8s
+Conformance <https://github.com/cncf/k8s-conformance>`__ which is
+expected to grow according to the ongoing requirement traceability.
+
+`End-to-End
+Testing <https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md>`__
+basically asks for focus and skip regexes to select or to blacklist
+single tests:
+
+-  focus basically matches Conformance or `Testing Special Interest
+   Groups <https://github.com/kubernetes/community/blob/master/sig-testing/charter.md>`__
+   in sub-sections below
+-  skip excludes the SIG labels listed as optional in `Reference
+   Architecture-2 (RA-2) Chapter
+   6 <https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/kubernetes/chapters/chapter06.md>`__
```

Comment:
At the end we should have relative links here. Maybe this will require some work after we merged the conversion pr-s of other docs.

Reply:
No. At the end we would have internal links or intersphinx mappings. But we do find a smooth path as proposed here.